src/instrumenter/annotations.ts

```diff
@@ -43,25 +43,31 @@ export function rangeToLocRange(start: number, length: number, contents: string)
 
 /**
  * Convert a line/column source range into an offset range
- *
- * @param r line/column source range
  */
 function rangeToOffsetRange(r: Range): OffsetRange {
     return [r.start.offset, r.end.offset - r.start.offset];
 }
 
+/**
+ * Convert a line/column source range into an offset range
+ */
+export function rangeToSrcTripple(r: Range, fileInd: number): SrcTriple {
```

Comment:
This seems to be a typo:
```suggestion
export function rangeToSrcTriple(r: Range, fileInd: number): SrcTriple {
```

Reply:
Good catch! Thanks @axic! Fixed in 9df996d